VenafiPS/Public/Get-TppAttribute.ps1

```diff
@@ -17,8 +17,9 @@ Path to the object to retrieve configuration attributes.  Just providing DN will
 To be deprecated; use -Path instead.
 Object Guid.  Just providing Guid will return all attributes.
 
-.PARAMETER AttributeName
```

Comment:
Won't this break existing modules that provide the AttibuteName parameter ?

Reply:
The parameter name is already Attribute, just fixing the help.